datasource/netcdf_groundwater.py

```diff
@@ -71,12 +74,19 @@ def find_aggregation_netcdf_gw(netcdf_file_path):
 
 
 class NetcdfGroundwaterDataSource(BaseDataSource):
+    """NetcdfGroundwaterDatasource provides an abstraction layer to query
+    result data from a 3Di simulation stored in a netcdf
+
+    It provides an abstraction so you don't need to worry about whether the
+    data is stored in the results_3di.nc or aggregate_results_3di.nc
+    """
+
     PREFIX_1D = "Mesh1D_"
     PREFIX_2D = "Mesh2D_"
     PREFIX_1D_LENGTH = 7  # just so we don't have to recalculate
```

Comment:
a) ik voel me veiliger met zoiets: 

```suggestion
    PREFIX_1D_LENGTH = len(PREFIX_1D)  # just so we don't have to recalculate
```

b) Wordt dit nog gebruikt? Verderop haal je een methode die deze prefixes stript weg, vandaar.

Reply:
a + b) Dit gaat heel binnenkort weg. In de volgende pull-request (die deze klasse verder refactored) heb ik het eruit gehaald. Ik wou het nu nog niet verwijderen want ik wist nog niet zeker of andere methodes er nog gebruik van maakten.

Dit is overigens ook een van de redenen waarom ik graag threedigrid meer wil gebruiken in de plugin, dan hoef je geen zorgen te maken voor allerlei prefixes :)